Framework/Support/LinkGenerator.php

```diff
@@ -100,6 +108,26 @@ public function url(
         return $basePath . "?" . http_build_query($args);
     }
 
+    /**
+     * Builds the controller part of the query string from the given segments.
+     *
+     * This method normalizes the controller segments by trimming whitespace and converting
+     * the first character of each segment to lowercase. It then joins the segments with
+     * slashes to form the controller path used in the URL.
+     *
+     * @param array $segments The segments representing the controller path.
+     * @return string The normalized controller path for the query string.
+     */
+    private function buildControllerQueryValue(array $segments): string
+    {
+        $normalized = array_map(
+            static fn($segment) => lcfirst(trim((string)$segment)),
+            array_filter($segments, static fn($segment) => trim((string)$segment) !== '')
+        );
+
+        return implode('/', $normalized ?: [$this->router->getControllerName()]);
```

Comment:
Zase ta ista kontola, tu to nemôže byť prázdne, navyše zase ti nepôjde mapovanie viacslovnych nazvov